packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -37,6 +50,7 @@ export class TestFixture {
   private responseModifiers: SearchResponseModifierPredicate[] = [];
   private returnError = false;
   private reflectStateInUrl = true;
+  private redirected = false;
```

Comment:
👀

Reply:
that is because typescript was complaining! it should start as `false`